src/eba_tips.eliom

```diff
@@ -117,19 +117,20 @@ let%shared block ?(a = []) ~name ~content () =
     if Stringset.mem name seen
     then Lwt.return None
     else begin
-      let close_button = Ot_icons.D.close () in
+      let box_ref = ref None in
+      let close = [%client (fun () ->
+        let () = match !(~%box_ref) with
+          | Some x -> Manip.removeSelf x
+          | None -> () in
+        Lwt.async (fun () -> set_tip_seen ~%name);
+        Lwt.return () : _ -> _) ] in
```

Comment:
Thanks, looks good now: `close` truly does do something asynchronous, as opposed to just scheduling another asynchronous task.